SU-HSDO/suhumsci
# Feat(STN-882):  remove icon feature flag

# READY FOR REVIEW

## Summary
Removed `experimental feature toggle` from Vertical Cards template. Updated Calendar/Location/Speaker icons to allow for better alignment/spacing.

## Notes:
- This PR address [STN-882](https://sparkbox.atlassian.net/browse/STN-882) & [STN-706](https://sparkbox.atlassian.net/browse/STN-706)
- You may need to run `drush cr` to see template updates.

## Need Review By (Date)
asap

## Urgency
medium

## Steps to Test
1. Visit https://sparkbox-sandbox.suhumsci.loc/events/vertical-date-stacked-cards
2. Confirm that Vertical Cards have icons Calendar/Location/Speaker whilst having the `experimental feature toggle` disabled for the theme.
3. Confirm that the icons are no longer misaligned.
4. Confirm that the text beside each icon is inline with its siblings.
5.  Switch themes and repeat previous steps.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
